### ISSUE TITLE
CODING: Develop the projects section on projects section 

### ISSUE THREAD

**Issue by alpha2320** [anitab-org/anitab-org.github.io#81]
### Description
As a developer,
I need the following section of mockup to be developed,
so that I can add it in my projects page :  https://xd.adobe.com/view/e86737c2-ff84-4e23-b4b1-e541508d5279-bd34/

### mocks

spacing and all details

![image](https://user-images.githubusercontent.com/62046625/86233427-04a9d300-bbb3-11ea-9e37-5f507af9db30.png)

![image](https://user-images.githubusercontent.com/62046625/86233574-3c187f80-bbb3-11ea-853b-259d17cc83f0.png)

![image](https://user-images.githubusercontent.com/62046625/86233617-50f51300-bbb3-11ea-990f-780bf6db37fd.png)

{font style-avenir color style-#103B81 Weight-medium}
### Acceptance Criteria

- [x] Follows all the design guidelines as mentioned and is responsive enough on large screen devices.
- [x] when clicked on ( view more ) all the listed projects should up .
- [x] the github and zulip redirect links should work in all  cards .

### Definition of Done

- [x]  All of the required items are completed.
- [x]  Approval by 2 mentor.

### Estimation
6 -7 hours

HAPPY CODING !! 😃

**Comment by alpha2320** [comment]
@sakshi1499 @annabauza @nandini45  please have a look 😺!

**Comment by Sonal240** [comment]
@alpha2320 you may remove the label once the issue is approved and add status available label thanks

**Comment by alpha2320** [comment]
sure 😄

**Comment by annabauza** [comment]
I left a comment on Zulip @alpha2320

**Comment by happycoder0011** [comment]
Can I work on it?

**Comment by navkumar258** [comment]
Is it still open, is someone working on it..? I am looking for my first contribution. thanks !

**Comment by Sonal240** [comment]
Hey @happycoder0011 @navkumar258 the issue is on hold as the design is under review...thanks

**Comment by nandini45** [comment]
@happycoder0011  are you still interested in working on this issue?

**Comment by happycoder0011** [comment]
Yes I am interested.

**Comment by PriViji** [comment]
I am interested. Is anyone working on this?

**Comment by nandini45** [comment]
already taken by someone.. you can look for some other issue @PriViji

**Comment by happycoder0011** [comment]
![image](https://user-images.githubusercontent.com/62816994/86619738-c2c8c480-bfd8-11ea-807e-2975acb7e13d.png)
![image](https://user-images.githubusercontent.com/62816994/86619774-ceb48680-bfd8-11ea-99fb-22d6111f8e9b.png)

**Comment by happycoder0011** [comment]
this is what I have designed till now keeping in mind the design guidelines. I am stuck since it's my first contribution. Kindly guide me on how should I go ahead. I am unable to make further development.
I didn't want to waste your time so I thought to reach out.

**Comment by PriViji** [comment]
Just give Padding size to the image container (div)

**Comment by nandini45** [comment]
@happycoder0011  i am sorry i missed your comment did u solve your issue?

**Comment by happycoder0011** [comment]
I tried i am unable to resolve the issue.
You can unassign me.I am unable to make progress.

**Comment by nandini45** [comment]
@happycoder0011 i think u were half way through your work, are u sure u dont want to work on it?

**Comment by nandini45** [comment]
and if you have any issue please tell us we will help you resolve it?

**Comment by happycoder0011** [comment]
I am missing on three major features:
1.The view more option for more cards to expand
2.The tags of the languages in each card 
3. In the design, it is mentioned 4 cards in a row but according to card dimension only three fits.

I could not figure out a way to add them. I am using material UI.
See if anything can be done.
Thanks.

**Comment by nandini45** [comment]
firstly if three cards are fitting.. no problem i will ask the team and update you about it. what can you do and what changes can be done.
then the tags i have shared a link check that out and tell me if it of any help to u. the tags have to be permanent so just dont use the close function (i will suggest use any three tag like for ex javascript python java and place them all in the cards).
https://www.youtube.com/watch?v=ha4xwcJXwow
then view more option for cards. i think we don't need that but let me ask Anna about it and if we need it i will tell you how. 
i personally think if you have done half the work you can complete it too. we will help you out at each step. just update us about it

**Comment by sakshi1499** [comment]
@happycoder0011 Any updates?

**Comment by happycoder0011** [comment]
yeah,working on it.Will update soon.

**Comment by happycoder0011** [comment]
Thanks for supporting.

**Comment by happycoder0011** [comment]
1. I am done with all the other aspects except the view more.I am attaching the screenshots.So if an issue kindly bring to my notice.
2. Should I make a pull request?
3. The cards on making the screen size to approx 600px ,is getting organized in single card per row.Anything that I should do for it.
![image](https://user-images.githubusercontent.com/62816994/87967519-9040c080-cadc-11ea-91c2-b2a887f8aceb.png)
![image](https://user-images.githubusercontent.com/62816994/87967537-98006500-cadc-11ea-9d9f-f4775eb46905.png)
![image](https://user-images.githubusercontent.com/62816994/87967586-a9e20800-cadc-11ea-8077-fe358e06db20.png)

**Comment by nandini45** [comment]
@happycoder0011  yes go ahead u can make a pr .
 nope no changes as far as i knw

**Comment by happycoder0011** [comment]
It would take me time to convert it into react native code.So unassign me.

**Comment by annabauza** [comment]
quick update card background color is `#E1E8FD`

**Comment by happycoder0011** [comment]
I won't be able to do it.kindly assign to someone else.

**Comment by happycoder0011** [comment]
You can take my work and someone can continue .

**Comment by annabauza** [comment]
can I make this , developer is not responding.

**Comment by PriViji** [comment]
Hi,

I wasn’t feeling well and I apologize for the late response.
I appreciate, if someone could take this one. I can do any bug fixing.

Thanks & Regards,
Priyanka.
